instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty/v4_1/ChannelFutureListenerInstrumentation.java

```diff
@@ -27,7 +27,7 @@
 
 import com.google.auto.service.AutoService;
 import io.netty.channel.ChannelFuture;
-import io.opentelemetry.auto.instrumentation.netty.v4_1.server.NettyHttpServerDecorator;
+import io.opentelemetry.auto.instrumentation.netty.v4_1.client.NettyHttpClientDecorator;
```

Comment:
i think this is a good catch, it seems to me this instrumentation only applies to netty client, should we move it to the client subpackage?